### ISSUE TITLE
version isort-4.3.0 isn't installable on RHEL7/Centos7

### ISSUE THREAD

**Issue by Pajinek** [timothycrosley/isort#658]
centos-release-7-4.1708.el7.centos.x86_64
python-2.7.5-58.el7.x86_64

```
>> pip install isort
Downloading/unpacking isort
  Downloading isort-4.3.0.tar.gz (54kB): 54kB downloaded
  Running setup.py egg_info for package isort
    /usr/lib64/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'python_requires'
      warnings.warn(msg)
    error in isort setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers
    Complete output from command python setup.py egg_info:
    /usr/lib64/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'python_requires'

  warnings.warn(msg)

error in isort setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers

----------------------------------------
Cleaning up...
Command python setup.py egg_info failed with error code 1 in /env/build/isort
Storing complete log in /root/.pip/pip.log
``

**Comment by alkovpro** [comment]
This bug had broke my whole docker ci setup %(
As a solution I had to add "isort==4.2.9" before "pylint" in my requirements files in several repos.

**Comment by coffeeismagic** [comment]
Same issue for Ubuntu 14.04.2. Had to restrict it to <4.3.0 in order for it to work.

**Comment by asottile** [comment]
Looks to be a duplicate of #652

**Comment by timothycrosley** [comment]
This should now be fixed in release 4.3.1, Thank you!
